.gitignore

```diff
@@ -82,6 +82,7 @@ secrets.tar
 # Python things
 __pycache__
 .python-version
+.venv/
```

Comment:
Selfishly added this to make things a little cleaner on my end - let me know if I should revert!

Reply:
Open a separate PR with that change

Reply:
coolio you got it

Reply:
https://github.com/vial-kb/vial-qmk/pull/325